COMMUNICATIONS.md

```diff
@@ -0,0 +1,94 @@
+# OpenSearch Project Communication 
+
+- [Overview](#overview)
+- [Public Slack](#public-slack)
+  - [Getting Started](#getting-started)
+  - [Workspace Channels](#workspace-channels)
+  - [Tips](#tips)
+
+## Overview
+
+The purpose of this document is to provide references and information regarding
+the communication channels for the OpenSearch Project. All communication is
+subject to the [OpenSearch Code of Conduct](CODE_OF_CONDUCT.md). Please see
+[CONTRIBUTING](CONTRIBUTING.md) if you're interested in contributing to the
+project.
+
+## Public Slack
+
+OpenSearch has a workspace on [Slack](https://opensearch.slack.com) to provide
+public communication channels for all people interested in OpenSearch.
+
+The following guidelines include a Getting Started for steps to register and
+setup the workspace along with tips for using the workspace. Please read through
+these guidelines carefully and thoroughly. All communications and behavior on
+the public slack workspace is subject to the 
+[OpenSearch Code of Conduct](CODE_OF_CONDUCT.md).
+
+### Getting Started
+
+_Join Us_
+
+  * Register your account at https://opensearch.slack.com. You will be required
+    to enroll in two-factor authentication to ensure the best possible security
```

Comment:
So, I could understand requiring maintainers to use 2FA for access to GH and making commits. And while I wholeheartedly endorse 2FA everywhere, this feels like friction and we're appealing to authority (security) to justify the friction. While I'd love everyone to be perfectly secure, I am not sure the juice is worth the squeeze. At a minimum, we should be looking for whether this is enough friction that folks don't continue with the sign up process.

Reply:
This is a good point. The 2FA intent was to minimize [Griefers](https://en.wikipedia.org/wiki/Griefer). Maybe enabling it is premature at this point since we're just starting to build a vibrant community? I'm certainly not opposed starting w/o 2FA and enabling if slack bombing becomes a problem. Curious what others think.

Reply:
I don't think of it as minimizing malicious folk causing pain by trolling, I see it as stopping malicious folk impersonating others if they're only relying on a weak password.

Reply:
Wholeheartedly agree with 2FA on both the intent of minimizing trolls as well as a step to hopefully stop anyone from maliciously impersonating someone else. In a perfect scenario, we would be able to have people sign up with their GitHub accounts to truly identify who's who between the different area. (something I'm likely going to implement on the forum as an option)